schemas/test/en/test_new_variants_content.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
These new schemas are just copies of the old ones with the when's updates, I have not necessarily reviewed them for content, although I did spot an error on one

Reply:
Some minor changes possibly needed, answer label shouldn't be a question. Should be something like the label for age answer in the next schema.